projects/storefrontlib/src/lib/user/facade/user.service.ts

```diff
@@ -1,78 +1,56 @@
 import { Injectable } from '@angular/core';
+
 import { Store, select } from '@ngrx/store';
-import { Observable } from 'rxjs';
 
-import * as fromStore from '../store';
 import {
   User,
   Order,
   PaymentDetails,
   Address,
   Region,
   Title,
-  Country
+  Country,
+  OrderHistoryList
 } from '@spartacus/core';
 
+import { Observable } from 'rxjs';
+
+import * as fromStore from '../store';
+
 @Injectable()
 export class UserService {
-  readonly user$: Observable<User> = this.store.pipe(
-    select(fromStore.getDetails)
-  );
-
-  readonly orderDetails$: Observable<Order> = this.store.pipe(
-    select(fromStore.getOrderDetails)
-  );
-
-  readonly orderList$: Observable<any> = this.store.pipe(
-    select(fromStore.getOrders)
-  );
-  readonly orderListLoaded$: Observable<boolean> = this.store.pipe(
-    select(fromStore.getOrdersLoaded)
-  );
-
-  readonly paymentMethods$: Observable<PaymentDetails[]> = this.store.pipe(
-    select(fromStore.getPaymentMethods)
-  );
-  readonly paymentMethodsLoading$: Observable<boolean> = this.store.pipe(
-    select(fromStore.getPaymentMethodsLoading)
-  );
-
-  readonly addresses$: Observable<Address[]> = this.store.pipe(
-    select(fromStore.getAddresses)
-  );
-  readonly addressesLoading$: Observable<boolean> = this.store.pipe(
-    select(fromStore.getAddressesLoading)
-  );
-
-  readonly titles$: Observable<Title[]> = this.store.pipe(
-    select(fromStore.getAllTitles)
-  );
-
-  readonly allDeliveryCountries$: Observable<Country[]> = this.store.pipe(
-    select(fromStore.getAllDeliveryCountries)
-  );
-
-  readonly allRegions$: Observable<Region[]> = this.store.pipe(
-    select(fromStore.getAllRegions)
-  );
-
   constructor(private store: Store<fromStore.UserState>) {}
 
-  loadUserDetails(userId: string) {
-    this.store.dispatch(new fromStore.LoadUserDetails(userId));
+  /**
+   * Returns a user
+   */
+  getDetails(): Observable<User> {
```

Comment:
Can we just use `get()`? I am not sure about it.

Reply:
Done